src/Controls/tests/DeviceTests/Controls.DeviceTests.csproj

```
@@ -19,6 +19,9 @@
 
   <ItemGroup>
     <MauiImage Include="Resources\Images\*" />
+    <MauiImage Remove="Resources\Images\red-embedded.png" />
+    <None Remove="Resources\Images\red-embedded.png" />
+    <EmbeddedResource Include="Resources\Images\red-embedded.png" />
```

@jstedfast try putting:
```suggestion
    <EmbeddedResource Include="Resources\Images\red-embedded.png" LogicalName="red-embedded.png" />
```